gulpfile.js

```diff
@@ -125,6 +125,7 @@ declareExtension('amp-story', '0.1', true);
 declareExtension('amp-selector', '0.1', true);
 declareExtension('amp-web-push', '0.1', true);
 declareExtension('amp-position-observer', '0.1', false);
+declareExtension('amp-date-picker', '0.1', true);
```

Comment:
That is correct. It would be possible to do it in the gulpfile, but I created the `.sh` scripts to quickly prototype to avoid the overhead of configuring and reconfiguring gulp code while developing and trying to figure out my approach. I agree that adding it to the gulpfile build process is the ideal strategy long-term.